skycfg.go

```diff
@@ -223,8 +223,8 @@ func loadImpl(ctx context.Context, opts *loadOptions, filename string) (starlark
 	var load func(thread *starlark.Thread, moduleName string) (starlark.StringDict, error)
 	load = func(thread *starlark.Thread, moduleName string) (starlark.StringDict, error) {
 		var fromPath string
-		if thread.TopFrame() != nil {
-			fromPath = thread.TopFrame().Position().Filename()
+		if thread.CallStack() != nil {
+			fromPath = thread.CallStack().String()
```

Comment:
`fromPath` is a file path, which is not what I would expect to get from `thread.CallStack().String()`.